docs/content/manual/manual.yml

```
@@ -3662,7 +3662,7 @@ sections:
       For example, with `-L$HOME/.jq` a module `foo` can be found in
       `$HOME/.jq/foo.jq` and `$HOME/.jq/foo/foo.jq`.
 
-      If `$HOME/.jq` is a file, it is sourced into the main program.
+      If `~/.jq` is a file, it is sourced into the main program.
```

Maybe we want some separate comment about windows here? do windows users usually known where `%USERPROFILE%/.jq` would be in the file system, give some hint? (i'm a windows noob so have no idea :) )

---

I could use plain language applicable to everyone, like this?:
```suggestion
      If `.jq` is a file in the user's home directory, it is sourced
      into the main program.
```
But my opinion is that `~` is going to be understandable to most Windows command-line users, especially those who would be using `jq`.

While historically a Unix-ism, and not expanded by the older CMD shell, it has been adopted by PowerShell.  Also, `~` expansion is touched on a few lines up from here in the model search path.

Let me know if you want me to change it.

---

I went ahead with the language change.

---

👍 either way works for me for me